src/html.rs

```diff
@@ -31,6 +31,45 @@ impl<'a> Meta<'a> {
     }
 }
 
+/// Rewrite root path URL of static files in `raw_html` with `cdn_url`.
+pub fn rewrite_html_cdn_url(raw_html: &[u8], cdn_url: &str) -> Result<Vec<u8>> {
```

Comment:
I suggest merging `rewrite_cdn_url` and `rewrite_base_url` into a single function for better performace.

Reply:
Please, Review this commit https://github.com/zineland/zine/pull/140/commits/259eb5630f33d9aa82f4f573c9840ce7f15cce75